apps/oc_chef_authz/src/oc_chef_authz_org_creator.erl

```diff
@@ -92,9 +92,9 @@
 
            %% users
            {add_acl,
-            [mk_tl(container, [cookbooks, data, nodes, roles, environments])],
+            [mk_tl(container, [cookbooks, data, nodes, roles, environments, policies])],
             [create, read, update, delete], [{group, users}]},
-
+           {add_acl, [{container, policies}], [read], [{group, clients}]},
```

Comment:
Any movement (hehe) on this? Is that an "eventually" thing? It'd totally suck to have this preview feature released and tell people that they can't use it if they're upgrading, unless the intent isn't even to tell people that they can test it out.